sdk/src/Core/Amazon.Runtime/Internal/Transform/UnmarshallerContext.cs

```diff
@@ -64,7 +64,7 @@ public byte[] GetResponseBodyBytes()
             }
             else
             {
-                return new byte[0];
+                return Array.Empty<byte>();
```

Comment:
Since we have to compile for NET Framework 3.5 and NET Framework 4.5 can we add preprocessor directives 
`#if NETCOREAPP3_1_OR_GREATER`  around the `Array.Empty` calls? 

`
#if NETCOREAPP3_1_OR_GREATER
return Array.Empty<byte>();
#else
return new byte[0];
`

Reply:
Why not simply shim Array.Empty?

Reply:
With that we can declare it once and use it consistsantly everywhere

Reply:
Creating a shim for .NET Framework works as well.

Reply:
Ok will do. Thanks for catching those. I usually only open the netstandard solution because everything is quite heavy weight.

Reply:
Pushed. Let me know what you think